Bugfix July25 Patch Day

BiccyThiccy opened the pull request:
Use these as you go please:

Patch Notes: https://github.com/deliciousmods/1956_beta/wiki/July-2025-Patch-Notes

Please briefly hover over any focus you modify, and if you make significant additions check the entire tree and if you find any, fix them or report them; if there are none, add the country here: https://docs.google.com/document/d/1w2dStPoE_LBmcskbwXRJ5pKvNAj06W3awRq6e_ssPP4/edit?tab=t.0

BiccyThiccy committed:
RAJ - Squash 1 Launch Error + Many Spammers - Porting other _economic.txt + character file changes while at it

1: RAJ characters have had their names made into loc keys as it's a vanilla change we don't need to maintain.

2: Fixed 1 launch error + many spammers as listed in commit title.

3: Used Winmerge to bring over most significant _economic.txt changes while making our file more similar to vanilla's. Some stuff was left out due to uncertainty that we want the features or that they won't cause unintended issues.

BiccyThiccy committed:
Fix AFG-DLC Tree Having Few MIL-Advisors Compared to Vanilla

- Some of this was due to a few characters not being in the history file, being unnoticed because they weren't making errors
- Some of this is from making some r56 guys work on DLC Afghanistan by rewriting/disabling visible triggers

Commit:
Fixed da dog

Commit:
RAJ Nat Spirit fixes

Commit:
some RAJ port fixes

BiccyThiccy commented on common/on_actions/r56_on_actions.txt:
Shouldn't this go in the focus_tree_selection file?

Xaelorichero-02 replied:
No. This is a compability check when you set Asia free.

BiccyThiccy committed:
AFG - Moving dlc focus tree checks to tree_selection on_action

BiccyThiccy committed:
RAJ - Combining Nehru's advisor roles (we can't have use a global flag for it in allowed blocks @Xaelorichero-02 )

Commit:
Aditya Dibabha, Phayom Chulanont portraits

BiccyThiccy committed:
ROM - Fix messed up offsets for those with show_obsolete_branch off + polishing stuff

Commit:
Romanian offset fix

Commit:
Mutually exclusives

BiccyThiccy committed:
Amend Romania offset fix

Commit:
quiet VSC

BiccyThiccy committed:
R56 Portugal AI Improvements

I noticed that due to some of their priority to do certain decisions, the AI wouldn't end up with enough political power to appoint mil high command when WW2 starts, or take advantage of some of the powerful late-game decisions. These decisions are also needed to get rid of the illiteracy penalties so this was hurting the AI's performance.

Secondarily, we now carry over vanilla's nerf to 'Old Guard' making the role much less useful. I have changed the AI's likelyhood to select Portugal's Old Guard advisor in favor of their Army Defense Expert and Army Fanaticism Expert.

These tweaks improve that and should make the AI usage of the focus tree a bit better.

BiccyThiccy committed:
JAP - Fix Missing LOC

Commit:
afg offsets

katsuragiyy committed:
Fixes

InfraTheIdiotic committed:
Update 397-Syktyvkar.txt

Fix missing victory point: Syktyvkar

InfraTheIdiotic committed:
Copied Khakas namelist from vanilla

InfraTheIdiotic committed:
Copied Seychellois namelist to Mauritius as it inaccurately used Arabic names

InfraTheIdiotic committed:
Fix annoying punctuation error in German focus name (Improve the Logistics System)

InfraTheIdiotic committed:
Fixed bad framing of Bhutanese neutral, democratic and communist flags

InfraTheIdiotic committed:
INS: Decision 'Expand the Bandoeng Arms Factory' now gives land warfare facilities

InfraTheIdiotic committed:
INS: Fixed decision 'Nationalize Artillerie Inrichtingen Bandoeng' requiring a Burmese state

marijn211 committed:
Merge branch 'Beta-Edits' into Bugfix-7-July-25

marijn211 commented on common/characters/AFG.txt:
```diff
@@ -293,7 +283,10 @@ characters = {
 				NOT = {
-					has_completed_focus = AFG_form_weekh_zalmian
+					if = {
+						limit = { has_global_flag = afg_dlc_focus_tree }
```
These halve empty if-else statements look quite odd

Xaelorichero-02 replied:
This looks completely normal

BiccyThiccy replied:
> These halve empty if-else statements look quite odd

I don't think we have an if check for the 56 tree, it looks a bit weird but having a NOT={ check for the dlc tree wouldn't look quite right either imo

It would look more normal if it was filled with stuff in the  if block

marijn211 replied:
> This looks completely normal

The highlighted part does, but there being nothing in the if-statement just below and only the else looks weird

marijn2112 commented on common/characters/AFG.txt:
```diff
@@ -522,7 +505,10 @@ characters = {
 				NOT = {
-					has_completed_focus = AFG_form_weekh_zalmian
+					if = {
+						limit = { has_global_flag = afg_dlc_focus_tree }
```
This looks odd

marijn211 commented on common/characters/AFG.txt:
```diff
@@ -765,22 +731,26 @@ characters = {
-							has_country_flag = AFG_dissatisfied_haji_mirzali_khan
+					}
+					else = {
+						AND = {
```
Doesn't have be adjust but be aware that inside `if` statements, `AND` is the default again, even if the scope outside it was `OR`
